src/implementations/twig/components/gallery/README.md

```diff
@@ -63,6 +64,12 @@ npm install --save @ecl/twig-component-gallery
   sr_video_player: 'Video player',
   items: [ 
     { 
+      picture: {
```

Comment:
it's almost irrelevant since this will be overridden in storybook with the real data, but it should be "thumbnail", right, not "picture"

Reply:
updated